src/vivarium_census_prl_synth_pop/constants/data_values.py

```diff
@@ -57,6 +57,7 @@
 #####################
 
 # data from https://www.jec.senate.gov/public/index.cfm/republicans/fl/
+BUSINESS_NAMES_MAX_TOKENS_LENGTH = 15
```

Comment:
it would be worth double checking with abie that this number wasn't chosen arbitrarily. a smaller max token will let it run faster